### ISSUE TITLE
Smart Connections - Obsidian GPT: "! Error talking to " message when trying to access notes

### ISSUE THREAD

**Issue by ConskiH** [brianpetro/obsidian-smart-connections#589]
Hi there

I am having trouble connecting the 'Smart Connect - Obsidian' GPT to my notes. Every time I ask a question relating to my notes, I get an 'Error talking to' message. This was working a few weeks back and I was really excited about the potential improvements to my workflow.

I have checked that my OpenAI account has a debit card attached and that the APIs I am using are valid. I have disconnected and reconnected the plugin.

Any suggestions on what to try next?

Many thanks.

<img width="1022" alt="Screenshot 2024-05-07 at 19 43 36" src="https://github.com/brianpetro/obsidian-smart-connections/assets/64075925/7b9b0289-5e79-409f-bcb5-91a206797945">

**Comment by brianpetro** [comment]
Hey @ConskiH 

Thanks for the screenshot. It makes this easy to solve 😊

The GPT in your screenshot is the original, which will be discontinued soon. The new GPT has many more features and prevents you from uploading your data to the cloud.

Here is the link to the new GPT that works with the Smart Connect software: https://chatgpt.com/g/g-9Xb1mRJYl-smart-connect-obsidian

Thanks for your support & using Smart Connections!
🌴

**Comment by brianpetro** [comment]
Hey @ConskiH 

With the new GPT, in the Smart Connect software which makes the GPT work, there is a "Open Console Logs" button at the bottom.

Any errors will be logged there. 

You can screenshot those errors and I will review them to let you know what's going on.

Thanks for your patience in figuring this out
🌴

**Comment by ConskiH** [comment]
Thanks again Brian.

Screenshots below.

All messages:
<img width="909" alt="Screenshot 2024-05-09 at 18 47 24" src="https://github.com/brianpetro/obsidian-smart-connections/assets/64075925/8545dc58-8771-434e-86f2-1bf774e76f72">

Errors expanded:
<img width="1113" alt="Screenshot 2024-05-09 at 18 47 44" src="https://github.com/brianpetro/obsidian-smart-connections/assets/64075925/43026b18-4d91-441d-b22f-69f80132078f">

Depreciated features expanded:
<img width="1114" alt="Screenshot 2024-05-09 at 18 47 59" src="https://github.com/brianpetro/obsidian-smart-connections/assets/64075925/d6e25be9-fe35-4506-8c17-164d5107ae95">

Response blocked expanded:
<img width="1106" alt="Screenshot 2024-05-09 at 18 48 14" src="https://github.com/brianpetro/obsidian-smart-connections/assets/64075925/96fb12a4-f8bc-48fd-bf9b-be5ad298a8c1">

**Comment by brianpetro** [comment]
Hey @ConskiH 

Thanks for the screenshots!

Unfortunately, I need you to do it again.

This time, click the "Console" tab that's to the left of the "Issues" tab in your screenshots.

Thanks for your help in figuring this out
🌴

**Comment by brianpetro** [comment]
Also, it looks like you might be screenshotting the console in Chrome for ChatGPT (based on some of the URLs in your screenshots).

Make sure to screenshot the console in the Smart Connect software so I can help 🌴

**Comment by ConskiH** [comment]
Hi Brian,

Thank you again for your help. I went back to basics and started from the very beginning. I installing the app with the settings as instructed in your screencast. I definitely hadn't followed every step correctly, and now that I have, everything is working as expected.

Thank you for your patience as I learned these new tools. I can already see this will be a game-changer for my research and personal note-taking. I look forward to learning more.

Thanks again,

Conor
